### DISCUSSION TITLE
更新日志

### DISCUSSION THREAD

**Discussion by share121** [share121/inter-knot#4590] [公告]
![微信图片_20240908100047](https://github.com/user-attachments/assets/c2556eca-df9e-43f8-8b92-4f9ced0fa388)

# 绳网开发计划

1. 还原界面
2. 添加小工具（如抽卡查询）
3. 优化性能

# 更新日志

## 2.14.2+21

1. 还原度更高

![1](https://github.com/user-attachments/assets/5a519971-3237-4673-a1c6-33aa5fcf4564)

## 2.14.1+19

1. 修复图标白边
2. 修改包名
3. 减小应用大小

## 2.14.0+18

1. 防止换页太快

## 2.13.0+17

1. 优化加载机制

## 2.12.1+16

1. 修复搜索页一直转圈的问题
2. 添加新协作者

## 2.12.0+15

1. 修复一直转圈问题

## 2.11.0+14

1. 实现登录功能

## 2.10.1+13

1. 修复举报链接跳转问题

## 2.10.0+12

1. 修复 Api 被人刷爆的问题

## 2.9.0+11

1. ⚠️ 重大更新，这个版本必须更新！！！！！

## 2.8.1+10

1. 降低获取举报频率

## 2.8.0+9

1. 完全实现分区功能

## 2.7.3+7

1. 修复手机端输入法自动弹出的问题

## 2.7.2+6

1. 浅色模式回归

## 2.7.1+4

1. 恢复举报

## 2.7.0+4

1. 移除登录

## 2.6.0+3

1. 修改文档链接（https://d.inot.top）

## 2.6.0+1

1. 支持切换 api

## 2.5.0+1

1. 添加登录页面

## 2.4.1+1

1. `绳网.top` 上线

## 2.4.0+1

1. 添加速率限制

## 2.3.3+1

1. 删除设置按钮

## 2.3.2+1

1. 修复 MacOS 平台安装界面背景图片显示异常的问题

## 2.3.0+1

适配 MacOS 平台（感谢 [@AetherLude](https://github.com/AetherLude)）

## 2.2.0+1

适配 Linux 平台

## 2.1.0+1

1. apk 签名
2. 修复手机端样式太挤
3. 更改举报区样式，更加友好
4. 修复 heroTag 重复导致的错误、卡顿
5. 修复加载评论区时获取滚动条位置引发的报错
6. 修改 FloatingActionButton 间距，更加美观

## 2.0.0+1

1. 修复网页端打不开的问题
2. 修复侧边栏 Discord 打不开的问题
3. 新增举报功能
4. 新增 IOS、MacOS 端

## 1.5.0+1

1. 实现历史功能

## 1.4.0+1

1. 实现收藏功能

## 1.3.0+2

1. 新增更新检测
2. 优化报错展示

## 1.2.0+1

1. 实现分区功能
2. 展示用户名而不是注册名

## 1.1.0+4

1. 实现了自动构建 windows 单文件版

**Comment by XixcQAQ** [comment]
好y

**Comment by Liuli-Lumine** [comment]
网页版访问不了了吗<img src="https://img-static.mihoyo.com/communityweb/upload/5c433e340acca5043629fc84299a326a.png" alt="三月七_生气" width="75" >

**Reply by HKLHaoBin** [reply]
网页版是可以访问的 但是要下载插件，总之就很麻烦，还是直接在手机或者用电脑软件端也不错

**Reply by jane-10086-jpg** [reply]
你好，我现在登录进去，但是无法发布内容，只找到评论区

**Reply by Palaluo** [reply]
o

**Comment by Emotional-Transparent** [comment]
小工具怎么用

**Reply by FAKERqx** [reply]
mm

**Comment by NyaiiSan** [comment]
做的确实牛逼哥们 ！ 

但是你的绳网Web图加载出来前的默认图片尺寸和加载出来后的图片尺寸不一致，看到一条逆天的就被其他加载出来的排版挤掉啦！ 
不能让它一直跳吧？

**Comment by Emotional-Transparent** [comment]
这是来自QQ邮箱的假期自动回复邮件。你好，我最近正在休假中，无法亲自回复你的邮件。我将在假期结束后，尽快给你回复。

**Comment by rtcrtc123** [comment]
啊？你的绳网是直接在github里面评论的啊。也是，就国内那备案没人能过。

**Comment by Time-Gardener** [comment]
up,你现在还有维护绳网吗，我刚刚进来，感觉好像人好少。up去哔哩哔哩推流一下，把更多绳匠捞进来说不定就热闹些了

**Comment by Emotional-Transparent** [comment]
这是来自QQ邮箱的假期自动回复邮件。你好，我最近正在休假中，无法亲自回复你的邮件。我将在假期结束后，尽快给你回复。

**Comment by Emotional-Transparent** [comment]
这是来自QQ邮箱的假期自动回复邮件。你好，我最近正在休假中，无法亲自回复你的邮件。我将在假期结束后，尽快给你回复。

**Comment by c328422** [comment]
20250423討論串的分類不見無法使用，現在已經無法發出新文章![Screenshot_20250422_163405_GitHub.jpg](https://github.com/user-attachments/assets/3e750d09-e1c2-42ca-bcd1-055df5c3d9b0)

**Comment by Emotional-Transparent** [comment]
这是来自QQ邮箱的假期自动回复邮件。你好，我最近正在休假中，无法亲自回复你的邮件。我将在假期结束后，尽快给你回复。

**Comment by Emotional-Transparent** [comment]
这是来自QQ邮箱的假期自动回复邮件。你好，我最近正在休假中，无法亲自回复你的邮件。我将在假期结束后，尽快给你回复。

**Comment by Emotional-Transparent** [comment]
这是来自QQ邮箱的假期自动回复邮件。你好，我最近正在休假中，无法亲自回复你的邮件。我将在假期结束后，尽快给你回复。

**Comment by qingyangprince** [comment]
大大看我，首先是一个问题：手机安卓端出现发帖的分类无法正常显示
还有就是我，想要参与该项目的研发，求求了

**Reply by qingyangprince** [reply]
还有一点忘了说，大大可以注册一个类似爱发电的账号，然后再我的里加一个支持😁，这可以帮到您和这个项目

**Reply by qingyangprince** [reply]
希望大家顶一顶，让大大看到

**Comment by c328422** [comment]
最後訊息：此項目已停止，只能使用留下的串進行討論
